crates/wasmer-pack/src/pirita.rs

```diff
@@ -39,22 +41,47 @@ fn libraries(webc: &WebC<'_>, fully_qualified_package_name: &str) -> Result<Vec<
     let Manifest { bindings, .. } = webc.get_metadata();
     let libraries = bindings
         .iter()
-        .map(|b| load_library(b, webc, fully_qualified_package_name))
+        .map(|b| load_library(b.clone(), webc, fully_qualified_package_name))
         .collect::<Result<Vec<_>, _>>()?;
 
     Ok(libraries)
 }
 
-fn metadata(fully_qualified_package_name: &str) -> Result<Metadata, Error> {
+#[derive(Debug, Serialize, Deserialize)]
+struct InternalPackageMeta {
+    description: String,
+}
+
+/// Retrieves the package description from the given `Manifest` object,
+/// If the `wapm` annotation is not found in manifest, returns an empty string.
+///
+/// # Arguments
+///
+/// * `manifest` - A reference to the `webc::metadata::Manifest` object containing the package metadata.
+///
+/// # Returns
+///
+/// A string representing the package description, or an empty string if the `wapm` annotation is not found.
+fn get_description_from_webc_manifest(manifest: &Manifest) -> String {
+    let wapm: Option<webc::metadata::annotations::Wapm> =
+        manifest.package_annotation("wapm").unwrap();
+    match wapm {
+        None => "".to_string(),
+        Some(wapm) => wapm.description,
+    }
+}
```

Comment:
This function is probably better written as the following one-liner:

```rust
use webc::metadata::annotations::Wapm;

m.package_annotation("wapm")
    .ok()
    .map(|Wapm { description, .. }| description)
```

<details>
<summary>(which can be inlined into `metadata()` if you want)</summary>

```rust
fn metadata(webc: &WebC<'_>, fully_qualified_package_name: &str) -> Result<Metadata, Error> {
    let (unversioned_name, version) = fully_qualified_package_name.split_once('@').unwrap();
    let package_name = unversioned_name
        .parse()
        .context("Unable to parse the package name")?;

    let mut metadata = Metadata::new(package_name, version);

    if let Ok(Wapm { description, .. }) = webc.manifest.package_annotation("wapm") {
        metadata = metadata.with_description(description);
     }

    Ok(metadata)
}
```
</details>

That way we don't crash when the `"wapm"` annotation is missing, and we return `Option<String>` rather than using an empty string.

I feel like if PyPI requires a description to be provided, we should generate an error at the top of `wasmer_pack::py::generate_python()` or fall back to the empty string by making [this `description` field](https://github.com/wasmerio/wasmer-pack/blob/74f7cd0e7f427c9c1c74ccba3e0c97546c859d62/crates/wasmer-pack/src/py/mod.rs#L290) required and doing `description.as_deref().unwrap_or_default()` [here](https://github.com/wasmerio/wasmer-pack/blob/74f7cd0e7f427c9c1c74ccba3e0c97546c859d62/crates/wasmer-pack/src/py/mod.rs#L256).

What are your thoughts?